warmup.py

```diff
@@ -225,7 +227,12 @@ class RadialDistribution:
 
     """
     def __init__(self, rmax, dr, rmin=0.0):
-        pass
+        self.rmax = rmax
+        self.dr = (rmax - rmin) / np.round((rmax - rmin) / dr)
+        self.rmin = rmin
+        self.edges = np.arange(rmin, rmax + (dr / 2), dr)
+        self.centers = np.arange(rmin + (dr / 2), rmax + (dr / 2), dr)
```

Comment:
Because floating-point math is imprecise, you should compute `centers` from `edges`.